dcs-dtc/Models/AH64/Waypoints/Waypoint.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+namespace DTC.Models.AH64.Waypoints
+{
+    public class Waypoint
+    {
+        private static Regex MGRSRegex = new Regex("^(\\d\\d\\s[A-Z][A-Z][A-Z]\\s\\d\\d\\d\\d\\s\\d\\d\\d\\d)$"); //MGRS format like 40 RDR 5123 6456

```

Comment:
Spaces should be optional and in different positions. My suggestion would be 
```
"^(\\d{2}\\s*[A-Z]\\s*[A-Z]{2}\\s*\\d{4}\\s*\\d{4})$"
```
`40RDR51236456` is just as valid as `40 R DR 5123 6456` or `40R DR12345678`

Reply:
replaced regex